test/parse.js

```diff
@@ -405,33 +415,40 @@ var tests = [{
     ]
 }, {
     message: 'should parse comments inside functions',
-    fixture: 'url( "/demo/bg.png" /*comment*/ )',
+    fixture: 'rgba( 0, 0, 0/*,.5*/ )',
     expected: [
-        { type: 'function', sourceIndex: 0, value: 'url', before: ' ', after: ' ', nodes: [
-            { type: 'string', sourceIndex: 5, value: '/demo/bg.png', quote:'"'},
-            { type: 'space', sourceIndex: 19, value: ' ' },
-            { type: 'comment', sourceIndex: 20, value: 'comment'}
-        ] }
+        { type: 'function', sourceIndex: 0, value: 'rgba', before: ' ', after: ' ', nodes: [
+            { type: 'word', sourceIndex: 6, value: '0' },
+            { type: 'div', sourceIndex: 7, value: ',', before:'', after: ' ' },
+            { type: 'word', sourceIndex: 9, value: '0' },
+            { type: 'div', sourceIndex: 10, value: ',', before:'', after: ' ' },
+            { type: 'word', sourceIndex: 12, value: '0' },
+            { type: 'comment', sourceIndex: 13, value: ',.5'}
+        ] }        
     ]
 }, {
-    message: 'should parse comments at the start of url function',
-    fixture: 'url( /*comment*/ /demo/bg.png )',
+    message: 'should parse comments inside url functions with qouted first argument',
+    fixture: 'url( /*before*/ "/demo/bg.png" /*after*/ )',
```

Comment:
Yes I [know](https://github.com/TrySound/postcss-value-parser/pull/22#discussion_r47449224), But it might be a good option if used by a post processor. Moreover unclosed functions are also invalid.

The decision is yours!